### PR TITLE
chore: Update dry run config with component-toolkit build step

### PR DESCRIPTION
**Pull request by pan-kot**
Required for: https://github.com/cloudscape-design/components/pull/714

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by just-boris** [comment]
This needs to be tested first. To do this, you need to to open a PR in components repository with changes like this: https://github.com/cloudscape-design/components/pull/717/files?w=1

In your case, the override branch will be `chore-update-dry-run-with-component-toolkit`

**Comment by pan-kot** [comment]
> chore-update-dry-run-with-component-toolkit

https://github.com/cloudscape-design/components/pull/723

**Comment by pan-kot** [comment]
Testing dry-run here: https://github.com/cloudscape-design/components/pull/723

File: .github/workflows/dry-run.yml [hunk 1]
```diff
@@ -72,6 +72,17 @@ jobs:
...
+  buildComponentToolkit:
+    name: Build component-toolkit
+    runs-on: ubuntu-latest
+    needs:
+      - buildBrowserTestTools
+    steps:
+      - uses: cloudscape-design/.github/.github/actions/build-package@main
+        with:
+          package: component-toolkit
+          artifact_path: theming-*.tgz
```
Comment (michaeldowseza) [inline .github/workflows/dry-run.yml]: Copy paste mistake?

Comment (pan-kot) [reply .github/workflows/dry-run.yml]: Looks like it - thanks!

**Review by just-boris — APPROVED** [review]

File: .github/workflows/dry-run.yml (hunk 1 above)
Comment (just-boris) [reply .github/workflows/dry-run.yml]: And it still passed the build with incorrect line. Interesting 🤔

**Pull request merged**